### PR TITLE
task-1 component warm up

### PR DESCRIPTION
**Pull request by filipHartman**

### CODE REVIEW

File: src/task-1/GenreSelect/GenreSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <label>{props.genre}</label>
+  </div>
+);
+
+interface IProps {}
+interface State {
+  isOpen: boolean;
+}
+
+export class GenreSelect extends Component<IProps, State> {
```
Comment (dragonflypl) [inline src/task-1/GenreSelect/GenreSelect.tsx:18]: Do you need this iprops repeated ? perpahs anonymous interface could be used?

File: src/task-1/Search/Search.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Search.css';
+import './../../styles/button.css';
+
+interface IProps {}
+
+const SearchInput = (props: { onChange: any }) => (
```
Comment (dragonflypl) [inline src/task-1/Search/Search.tsx:7]: Maybe try to keep components per file, even smallest

**Review by dragonflypl — APPROVED** [review]

File: src/task-1/GenreSelect/GenreSelect.tsx (hunk 1 above)
Comment (filipHartman) [reply src/task-1/GenreSelect/GenreSelect.tsx]: fixed

File: src/task-1/Search/Search.tsx (hunk 2 above)
Comment (filipHartman) [reply src/task-1/Search/Search.tsx:5]: ok will do in next tasks

**Pull request merged**